app/controllers/estimates_controller.rb

```diff
@@ -51,6 +51,11 @@ def download
     end
   end
 
+  def external_cw_form_redirect
```

Comment:
As this method is about sending the user to an external document, I think the natural place for it is in `DocumentsController`, which does exactly that. You could even reuse the `show` method, i.e. if the URL was `/documents/cw_forms?referrer=view_results&assesment_code=foo` then (so long as you added `cw_forms` to GuidanceLinkService), the link tracking would be handled for you automatically by the existing code. But equally it's not really a guidance link so you could argue that it's better to give it its own endpoint.

(BTW, so long as the analytics event you use starts with `click_`, the [existing link click Blazer query](https://check-your-client-qualifies-for-legal-aid.service.gov.uk/data/queries/24-guidance-link-clicks) will show up clicks alongside guidance link clicks, which is potentially quite useful

Reply:
I did wonder about just tapping into the existing stuff in DocumentsController. I was just a bit reluctant as I didn't think the external webpage was like a 'document' as such (but I guess it kind of is) and, as you've said, it's not a guidance link which is why I ended up making a new method. BUT it would be nice to reuse stuff so maybe I can play around with a bit of renaming.

Reply:
Also that's great news about the Blazer query 🙌

Reply:
Yeah, both DocumentsController and GuidanceLinkService are potentially badly named, because some of the links are to web pages with legislation, which are neither "documents" in the typical sense nor "guidance". So I'd definitely 👍 some renaming!